source/1_engine/rex_engine/include/rex_engine/diagnostics/logging/logger_config.h

```diff
@@ -6,6 +6,6 @@ namespace rex
 {
   namespace diagnostics
   {
-    void load_log_levels(const rsl::vector<rsl::string_view>& args);
+    void init();
```

Comment:
I don't think we should adjust the name of this function ... init does not tell me a lot. Load log levels explains what the function does.

Reply:
But it's not loading them, it's initialising them.

Reply:
Then you might call it initialize log levels, diagnostics::init() is a bbit generic